src/ngraph/runtime/cpu/pass/cpu_mkldnn_primitive_build.cpp

```diff
@@ -2400,7 +2400,7 @@ namespace ngraph
                     writer << "*cg_ctx->mkldnn_descriptors[" << desc_index + 2 << "]);\n";
 
                     writer << "\nmkldnn::post_ops ops;\n";
-                    if (has_relu<OP>(node))
+                    if (false)
```

Comment:
similar to the`#if 0`, here is some relu related code, which is dead-code right now